dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -186,7 +186,7 @@ public <T> T getNonReloading(Class<T> clazz, ServiceConfiguration serviceConf) {
         Channel channel = new LiveReloadingChannel(
                 DnsSupport.pollForChanges(
                                 params.dnsNodeDiscovery(),
-                                DnsPollingSpec.SERVICE_CONFIG,
+                                DnsPollingSpec.serviceConfig(channelName),
```

Comment:
I updated these so that we'll get more specific metric tags describing ongoing work, and used the info within the background DNS polling task failure logging so we don't have to guess about the unsafe URI.